package.json

```diff
@@ -120,5 +122,19 @@
     "> 1%",
     "last 2 versions",
     "not ie <= 8"
-  ]
+  ],
```

Comment:
Please remove all additions in package.json except the additions of the dependencies.

Reply:
Ok, will remove all additions

Reply:
I also love to contribute in common creative org

Reply:
what you going to do about simplebar package, does @brenoferreira  approve it.

Reply:
If @brenoferreira approve please merge it.

Reply:
if you want any change please inform me.